pkg/gui/controllers/helpers/confirmation_helper.go

```diff
@@ -228,6 +228,32 @@ func (self *ConfirmationHelper) CreatePopupPanel(ctx goContext.Context, opts typ
 	return self.c.PushContext(self.c.Contexts().Confirmation)
 }
 
+func underlineLinks(text string) string {
+	result := ""
+	remaining := text
+	for {
+		linkStart := strings.Index(remaining, "https://")
+		if linkStart == -1 {
+			break
+		}
+
+		linkEnd := strings.IndexAny(remaining[linkStart:], " \n>")
+		if linkEnd == -1 {
+			linkEnd = len(remaining)
+		} else {
+			linkEnd += linkStart
+		}
+		underlinedLink := style.AttrUnderline.Sprint(remaining[linkStart:linkEnd])
+		if strings.HasSuffix(underlinedLink, "\x1b[0m") {
+			// Replace the "all styles off" code with "underline off" code
+			underlinedLink = underlinedLink[:len(underlinedLink)-2] + "24m"
+		}
```

Comment:
This is pretty hacky, but we need it so that we don't cancel the bold text. I don't see a better way of doing this except by making changes to `gookit/color`.